### PR TITLE
update postgres initialization for compose/django.md

### PR DESCRIPTION
**Pull request by snoop2head**
## Proposed changes

Updated postgres initialization for compose/django.md
* Fixed issue #10532 
* Added environment arguments to db in docker-compose.yml
* Added password argument in django project's settings.py

## Related issues 

[Issue #10532 django quickstart fail "Database is uninitialized and superuser password is not specified."](https://github.com/docker/docker.github.io/issues/10532#event-3225064933)

### CODE REVIEW

**Comment by usha-mandya** [comment]
Thank you for the update, @snoop2head.

@ulyssessouza Could you PTAL?

File: compose/django.md [hunk 1]
```diff
@@ -60,23 +60,31 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
-        depends_on:
-          - db
+    # docker-compose.yml
+    
+     version: '3'
+    
+     services:
+       db:
+         image: postgres
+         ports:                       #add line.
```
Comment (ulyssessouza) [inline compose/django.md:73]: This block can be ommited. The port is not exposed

File: compose/django.md [hunk 2]
```diff
@@ -60,23 +60,31 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
-      web:
-        build: .
-        command: python manage.py runserver 0.0.0.0:8000
-        volumes:
-          - .:/code
-        ports:
-          - "8000:8000"
-        depends_on:
-          - db
+    # docker-compose.yml
```
Comment (ulyssessouza) [inline compose/django.md:66]: Please, remove this comment line. The previous steps already mention the filename.

File: compose/django.md [hunk 3]
```diff
@@ -138,21 +146,24 @@ In this section, you set up the database connection for Django.
...
+2. Replace the `DATABASES = ...` with the following:
+
+       # setting.py
+       
+       DATABASES = {
+           'default': {
+               'ENGINE': 'django.db.backends.postgresql',
+               'NAME': 'postgres',
+               'USER': 'postgres',
+               'PASSWORD': 'postgres',  #add line.
```
Comment (ulyssessouza) [inline compose/django.md:158]: Please, remove the comment "#add line" from it

**Comment by usha-mandya** [comment]
Thank you @ulyssessouza.

@snoop2head Could you address the feedback from Ulysses? Thanks.

File: compose/django.md (hunk 2 above)
Comment (snoop2head) [reply compose/django.md:66]: Okay

File: compose/django.md (hunk 1 above)
Comment (snoop2head) [reply compose/django.md:73]: Just checked! Yes it can be omitted

File: compose/django.md (hunk 3 above)
Comment (snoop2head) [reply compose/django.md:158]: Roger

**Comment by snoop2head** [comment]
@ulyssessouza @usha-mandya 
Thank you for the feedback! 
* Omitting comments with file names & "add lines"
* Deleting uncompleted postgres port exposure code block

After reflecting changes from feedbacks, I ran the django tutorial from the beginning. 
Django server is working fine, with the following console log:

```shell
db_1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
db_1   | 
db_1   | 2020-04-15 17:06:40.956 UTC [1] LOG:  starting PostgreSQL 12.2 (Debian 12.2-2.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
db_1   | 2020-04-15 17:06:40.957 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1   | 2020-04-15 17:06:40.957 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db_1   | 2020-04-15 17:06:40.959 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1   | 2020-04-15 17:06:40.972 UTC [26] LOG:  database system was shut down at 2020-04-15 17:06:38 UTC
db_1   | 2020-04-15 17:06:40.976 UTC [1] LOG:  database system is ready to accept connections
web_1  | Watching for file changes with StatReloader
web_1  | Performing system checks...
web_1  | 
web_1  | System check identified no issues (0 silenced).
web_1  | 
web_1  | You have 17 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
web_1  | Run 'python manage.py migrate' to apply them.
web_1  | April 15, 2020 - 17:06:41
web_1  | Django version 2.2.12, using settings 'composeexample.settings'
web_1  | Starting development server at http://0.0.0.0:8000/
web_1  | Quit the server with CONTROL-C.
```

**Pull request merged**